submodules/wizard/wizard.js

```diff
@@ -151,17 +151,37 @@ define(function(require) {
 								callback: waterfallCallback
 							});
 						},
-						function(parentAccount, waterfallCallback) {
+						function getResellerAccount(parentAccount, waterfallCallback) {
 							var results = {
 									parentAccount: parentAccount,
 									servicePlans: []
 								},
 								resellerAccountId = self.wizardGetResellerAccountId(parentAccount);
 
-							self.wizardSetStore('resellerAccountId', resellerAccountId);
+							self.wizardRequestGetAccount({
+								accountId: resellerAccountId,
+								callback: function(err) {
+									var status = _.get(err, 'status'),
+										isResellerUnavailable = _.includes([ 403, 404 ], status);
```

Comment:
It is because I want to default to the parent account only when the reseller account no longer exists, or the access is forbidden. Other type of error may indicate that something is wrong, in which case it will be let to fail in the following requests.

Reply:
Gotcha